### PR TITLE
Issue 88 Added "Thick" Male Body For Run, Eliza's New Combat Animations, and Mapped Eyes to All Frames

### PR DESCRIPTION
**Pull request by jrconway3**
I currently have the assets split between "original" and "revised". I'm thinking of cutting it back to just "revised" and removing all the other assets, but let me know what you think.

In this set I went ahead and added the Revised Run assets for the standard "Male" run animation. I also added in all of Eliza's other animations, which also expanded the size of the ULPC again slightly, but I figured I'd better just do this now rather than have to do it again later. That way as I'm adding all of Eliza's animations I don't have to backtrack and add the combat animations later.

I did combine her two full slash sheets into one section, though. The Backslash animation is 11 frames long, but it shares the first 6 frames with Slash. As such, I kept the 7th frame of Slash, and just have a custom cycle for Backslash to skip the 7th frame.

I'm not ready for Eliza's outfits yet because I'm still missing all of the base frames for Cast, Thrust, Shoot, and Down. I'll be utilizing some help from the ULPC assets (like Longsleeve, etc) in order to fill out these missing arm frames, but I'm not quite ready for that yet.

Hairstyles should be much easier, I might just drop all those next including my own. The only hair frames I'm missing for my own assets are the new combat animations and emote, and I should have enough frames to implement those just fine. I'm also going to try and get the ears (including Makrohn's, which I'll map to all frames) and Tails as these should be relatively easy to fill out on the remaining frames, compared to clothing which is much harder.

I also went ahead and already mapped the Eye frames to all frames. I also did Cyclops Eyes, but in this case I redesigned them to have shading and a transparent border so they look a bit more natural... this one has been recolored to all the standard eye color palettes, but I kept the original one with flat colors as well as an option, though.

### CODE REVIEW

File: palettes/ulpc-cloth-palettes.json [hunk 1]
```diff
@@ -1,177 +1,227 @@
 {
     "brown": [
+        "#1d131e",
+        "#1d131e",
```
Comment (sanderfrenken) [inline palettes/ulpc-cloth-palettes.json:4]: Is it by intention that the first two entries of each palette array are the same? (except for the source)

**Comment by sanderfrenken** [comment]
`I currently have the assets split between "original" and "revised". I'm thinking of cutting it back to just "revised" and removing all the other assets, but let me know what you think.`

This remark applies to the run animation solely right? as in all the other animations are either new (climb, idle, slash new, etc) or just the original (magic, thrust, walk, thrust, slash, shoot, hurt). 

If that is true, I would indeed use just the revised images. That will not impact the original assets (backwards compatibility) and at the same time we set a new standard (revised) for the new animations.

**Comment by jrconway3** [comment]
Yeah, Run is the only thing that's different between these animations. I mostly stuck with Revised Run because Eliza has more outfits mapped to these animations (and outfits are by far the hardest things to configure), but also I just think this looks better.

Run is the only animation with significant changes. Most of the leg positions are the same, but the body moves more to make it feel like a more natural run.

Anyway, I'll go ahead and remove the default run animations now and only stick with Eliza's Revised. So now there won't be two sets of human heads and bodies.

**Comment by jrconway3** [comment]
Forgot to commit the index.html after I ran the update.

Let me know how the male run animations look. I had to carefully gauge how to handle them, but I think they look decent. These are combinations of Eliza's Masculine, Thin (converted to "Teen") alongside the original Male Run animation. The legs were in almost all cases the same size and shape, but in some cases I combined both leg positions to make the span longer. I tried to keep the narrower side run from Eliza, but I made the south/north animations still be closer to the width of the standard male walk, but not quite as wide because the body is moving more.

**Review by sanderfrenken — APPROVED** [review]

File: palettes/ulpc-cloth-palettes.json (hunk 1 above)
Comment (jrconway3) [reply palettes/ulpc-cloth-palettes.json:4]: It is intentional but it was accidentally added to the commit. Eliza's palettes were using two very similar colors along the border of the clothing so I was just simplifying it. However, I didn't even include the clothing conversions in this PR anyway so IDK how this change got here.

Yeah, double-checking it:
2a1722
271920


When I look up these two colors online I can't even tell the difference. The standard cloth palettes don't have two colors here either so I can't easily replace them with another similar color, so I just dropped the extra color.

**Pull request merged**